ssl/ssl_ciph.c

```diff
@@ -101,10 +101,7 @@ static const ssl_cipher_table ssl_cipher_table_cipher[SSL_ENC_NUM_IDX] = {
     {SSL_CHACHA20POLY1305, NID_chacha20_poly1305},
 };
 
-static const EVP_CIPHER *ssl_cipher_methods[SSL_ENC_NUM_IDX] = {
-    NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
-    NULL, NULL
-};
+static const EVP_CIPHER *ssl_cipher_methods[SSL_ENC_NUM_IDX];
```

Comment:
Speaking of irony. I'm not 100% sure why it's explicitly initialized with NULLs, but I do recall problem with Solaris long time ago, which was attributed to failure to initialize static variables to zero under some circumstances. What I'm saying is that this modification should be omitted too, but not necessarily because Sun C failed to comply with standard long time ago, but because it's unrelated modification.

On related note. As curious fact, i.e. not related to the above, because ssl_cipher_methods itself is not constant. But `const int foo[4];` and `const int foo[4] = {};` are compiled differently and end up either in .bss or .rodata segments.